Using glyph references

jishnu7 opened the pull request:
Glyphs can be optimized more. This PR doesn't cover possible references.

jishnu7 committed:
adding build instructions and gitignore

jishnu7 committed:
[Rachana] using k1 as reference

for ക്ട ക്ടു ക്ടൂ ക്ട്ര ക്ട്രു ക്ണ ക്ന ക്നു ക്നൂ ക്മ ക്മൊ ക്മൂ ക്സ ക്സു ക്സൂ ക്സൃ ക്റ്റ ക്റ്റു ക്റ്റൂ

jishnu7 committed:
[Rachana] using ch1 as reference

jishnu7 committed:
[Rachana] using nh as reference

jishnu7 committed:
[Rachana] using k3 as reference

jishnu7 committed:
[Rachana] Using k2, t3 as reference in similar glyphs

jishnu7 committed:
[Rachana] using th1 and tha2 as references in similar glyphs

jishnu7 committed:
[Rachana] using more glyphs as references

jishnu7 commented:
@santhoshtr @rajeeshknambiar

santhoshtr commented on README.md:
```diff
@@ -10,3 +10,12 @@ The source code repository of malayalam unicode fonts developed or maintained by
 * Suruma
+
+How to Build
+=====
```
We have a makefile that does all these. Following targets are already defined: default, clean, compile, install, uninstall, test,  webfonts, sdist

jishnu7 replied:
removed.

rajeeshknambiar replied:
Nice!

Has this been tested for regression? i.e, all changed glyphs?

On Sat, Sep 27, 2014 at 6:56 PM, Jishnu Mohan notifications@github.com
wrote:

> In README.md:
> 
> > @@ -10,3 +10,12 @@ The source code repository of malayalam unicode fonts developed or maintained by
> > - Rachana
> > - Raghu Malayalam Sans
> > - Suruma
> >   +
> >   +How to Build
> >   +=====
> 
> removed.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/smc/fonts/pull/8/files#r18124608.

## 

Rajeesh

jishnu7 committed:
no need of build instructions, make file already has that

rajeeshknambiar commented:
Reviewed the changes, looks good!
Could you update following glyphs in similar fashion?

k1nhu1, k1nhu2
k1shnh, k1shnhu1, k1shnhu2
k1shm1, k1shm1u1, k1shm1u2
k1shm1r1
th1s1th2, th1s1th2u1, th1s1th2u2
th1s1th2r1
th1s1n1, th1s1n1u1, th1s1n1u2
th1s1m1, th1s1m1u1, th1s1m1u2, th1s1m1r1
th4n1, th4n1u1, th4n1u2
th4m1, th4m1u1, th4m1u2
th4m1r1
s1k1, s1k1u1, s1k1u2
s1k1r1
s1k1k1, s1k1k1u1, s1k1k1u2
s1k1k1r1
s1k1r3, s1k1r3u1, s1k1r3u2
s1k2, s1k2u1, s1k2u2
s1t1, s1t1u1, s1t1u2
s1t1r3, s1t1r3u1, s1t1r3u2
s1th1, s1th1u1, s1th1u2
s1th1r1
s1th1r2
s1th1r3, s1th1r3u1, s1th1r3u2
s1th2, s1th2u1, s1th2u2
s1th2r1
s1n1, s1n1u1, s1n1u2
s1p1, s1p1u1, s1p1u2
s1p1r1
s1p1r3, s1p1r3u1, s1p1r3u2
s1p2, s1p2u1, s1p2u2
s1m1, s1m1u1, s1m1u2
s1m1r1
s1l3, s1l3u1, s1l3u2
s1s1, s1s1u1, s1s1u2
s1s1r1
s1s1r3, s1s1r3u1, s1s1r3u2
s1rhrh, s1rhrhu1, s1rhrhu2, s1rhrhr3, s1rhrhr3u1, s1rhrhr3u2
zhk1k1, zhk1k1u1, zhk1k1u2
zhn1n1u2

jishnu7 committed:
[Rachana] using k1sh as reference

jishnu7 committed:
[Rachana] Using th1s1 as reference

jishnu7 commented:
added k1sh\* and th1s1*.

ക il k1nhu1 and k1nhu2 slightly different than k1. and th4 and ധ in th4\* are slightly different, same for s1\* and zhk*.

Since these are very slight differences, I'm not sure whether these are intentional or not.

rajeeshknambiar commented:
I couldn't distinguish the difference (don't trust my eyes, though) and most probably such change if exists is unlikely intentional.

jishnu7 committed:
[Rachana] Using xx as reference

jishnu7 committed:
revert file creation time change

My fontforge installation modified file creation time.
This commit is to revert that change.

rajeeshknambiar commented:
Jishnu, were those changes made and ready for merge?

rajeeshknambiar commented:
I have merged the initial pull request and pushed out to savannah. Could you rebase the next changes on top of that?

jishnu7 commented:
Sorry for the late reply. 

I didn't get time to make other changes. I'll do it as a different PR. 

Updated github with latest changes.
